internal/sdkv2provider/resource_cloudflare_workers_kv.go

```diff
@@ -101,14 +101,15 @@ func resourceCloudflareWorkersKVDelete(ctx context.Context, d *schema.ResourceDa
 }
 
 func resourceCloudflareWorkersKVImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	namespaceID, key, err := parseId(d.Id())
-	if err != nil {
-		return nil, err
+	parts := strings.SplitN(d.Id(), "/", 3)
+	if len(parts) != 3 {
+		return nil, fmt.Errorf("invalid id (\"%s\") specified, should be in format \"accountID/namespaceID/keyName\"", d.Id())
 	}
 
-	d.Set("namespace_id", namespaceID)
-	d.Set("key", key)
-
+	d.Set(consts.AccountIDSchemaKey, parts[0])
+	d.Set("namespace_id", parts[1])
+	d.Set("key", parts[2])
+	d.SetId(fmt.Sprintf("%s/%s", parts[1], parts[2]))
```

Comment:
I see a similarity with https://github.com/cloudflare/terraform-provider-cloudflare/blob/483e9b5e5dc294d3ad57ee1dd0cf976af5b6b67d/internal/sdkv2provider/resource_cloudflare_workers_kv.go#L74 and wonder if it's worth having a `CreateId` function. Thoughts?

Reply:
I did it anyway. It's in a separate commit so I can remove if you think it's inappropriate.